### PR TITLE
Support for Ubuntu 16.04

### PR DESCRIPTION
**Pull request by traw**
**Description**

[] It needs and includes Unit Tests

**Changes in Client or Server public APIs**

[] It includes documentation for these changes in `/doc`.

### CODE REVIEW

**Comment by jcague** [comment]
please, update also the Dockerfile to use Ubuntu 16.04, it won't pass the tests otherwise.

**Comment by kekkokk** [comment]
You also need my patch to make libav compile in ubuntu 16 https://github.com/kekkokk/licode/commit/edb7e80e6d2df98dc0b1342c0819bdf8793dcb5b

File: erizo/src/CMakeLists.txt [hunk 1]
```diff
@@ -57,7 +57,7 @@ add_custom_target(lint
 #global variable
 set(THIRD_PARTY_INCLUDE "${CMAKE_CURRENT_SOURCE_DIR}/../../build/libdeps/build/include/")
 set(THIRD_PARTY_LIB "${CMAKE_CURRENT_SOURCE_DIR}/../../build/libdeps/build/lib/")
-
+include_directories(${THIRD_PARTY_INCLUDE})
```
Comment (jcague) [inline erizo/src/CMakeLists.txt]: why do we need to add the includes here?

File: erizo/generateEclipseProject.sh [hunk 2]
```diff
@@ -6,9 +6,9 @@ BIN_DIR="build"
 if [ -d $BIN_DIR ]; then
   cd $BIN_DIR
   # Set to Debug to be able to debug in Eclipse
-  cmake -G"Eclipse CDT4 - Unix Makefiles" -D CMAKE_BUILD_TYPE=Debug ../src
+  cmake -G"Eclipse CDT4 - Unix Makefiles" -D ERIZO_BUILD_TYPE=debug ../src
```
Comment (jcague) [inline erizo/generateEclipseProject.sh]: a more complete solution has already been merged and it is conflicting with this one, so please remove these changes

**Comment by kekkokk** [comment]
@traw can you please review the pr?

**Comment by lodoyun** [comment]
Thanks for starting this! We've got a new PR with the required changes plus docker support in #1300

**Pull request closed without merging**